tools/osemosys/features.yaml

```diff
@@ -0,0 +1,401 @@
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#reserve-margin-constraints
+      value: y
+    two_stage_stochastic:
+      source: []
+      value: n
+    voll:
+      source: []
+      value: n
+  spatial__resolution__simplification:
+    network:
+      source: []
+      value: n
+    resampling:
+      source: 
+        - https://github.com/KTH-dESA/GSA_Spatial_temporal
+        - https://doi.org/10.1016/j.isci.2025.111792
+      value: dev
+  system__carriers:
+    coupled:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+      value: y
+    decoupled:
+      source: []
+      value: n
+  system__limits:
+    capacity:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#new-capacity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#total-capacity-constraints
+      value: y
+    operation:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#annual-activity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#total-activity-constraints
+      value: y
+  time__foresight:
+    myopic:
+      source:
+        - https://github.com/KTH-dESA/OSeMOSYS_step
+        - https://doi.org/10.1186/s13705-024-00469-w
+      value: dev
+    perfect:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#objective-function
+      value: y
+  time__horizon:
+    multi_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    single_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    sub_day:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    sub_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+  time__resolution__simplification:
+    distortion_mitigation:
+      source: []
+      value: n
+    load_duration_curves:
+      source: []
+      value: n
+    resampling:
+      source: []
+      value: n
+    typical_periods:
+      source:
+        - https://doi.org/10.1016/j.enpol.2011.06.033
+      value: y
```

Comment:
Is it possible within OSeMOSYS to provide a full 8760 and have it sliced on your behalf? Or to provide the full 8760 and the slicing you want? If yes, this feature is valid, if not I'd say it is an `n`. It's one I need to consider separating out and asking "what temporal resolutions can be handled in the math formulation?" since some tools won't be able to handle an input of time slices.